front/components/assistant_builder/AssistantBuilderDataSourceModal.tsx

```diff
@@ -77,9 +80,28 @@ export default function AssistantBuilderDataSourceModal({
       variant="side-md"
       title="Manage data sources selection"
     >
-      <div className="w-full pt-12">
+      <div className="flex w-full justify-end pl-12 pr-2 pt-4">
```

Comment:
This is orthogonal to the vault effort, right? Instead of simulating those clicks we usually push the highest parent(s) in the array of selected resources, can we follow this existing pattern?

Reply:
>This is orthogonal to the vault effort, right ?

It's in the figma.

I don't understand "usually push the highest parent(s) in the array of selected resources"

Can you point me to some code ?